config/control-plane/kustomization.yaml

```diff
@@ -68,6 +68,9 @@ patches:
       - op: add
         path: /spec/template/spec/containers/0/args/-
         value: --is-kyma-managed
+      - op: add
```

Comment:
Note to reviewer: This will not be introduced here. This will be added to the landscapes in the management charts repo.

So when the review for this PR is finalized, the steps to merge it will be as follows:
- introduce a PR to lifecycle manager that only adds the flag without references or any other code
- add the flag value in the management charts repo
- Add the rest of the code here in another PR